specification/quota/resource-manager/Microsoft.Quota/preview/2023-06-01-preview/groupquota.json

```diff
@@ -0,0 +1,1797 @@
+        "operationId": "GroupQuotaSubscriptions_Delete",
+        "x-ms-examples": {
+          "GroupQuotaSubscriptions_Delete_Subscriptions": {
+            "$ref": "./examples/GroupQuotas-Subscriptions/DeleteGroupQuotaSubscriptions.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/MgIdInParameters"
+          },
+          {
+            "$ref": "#/parameters/GroupQuotaNameInParameters"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK."
+          },
+          "204": {
+            "description": "No Content."
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "GroupQuotas-Subscriptions"
+        ],
+        "summary": "Checks if the subscription is in GroupQuotas SubscriptionIdList.",
+        "description": "Checks if the subscription is in GroupQuotas SubscriptionIdList.",
+        "operationId": "GroupQuotaSubscriptions_Get",
+        "x-ms-examples": {
+          "GroupQuotaSubscriptions_Get_Subscriptions": {
+            "$ref": "./examples/GroupQuotas-Subscriptions/GetGroupQuotaSubscriptions.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/MgIdInParameters"
+          },
+          {
+            "$ref": "#/parameters/GroupQuotaNameInParameters"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/GroupQuotasEntity"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Management/managementGroups/{mgId}/{resourceProviderScope}/providers/Microsoft.Quota/groupQuotas/{groupQuotaName}/groupQuotaLimits/{resourceName}": {
+      "put": {
+        "tags": [
+          "GroupQuotaLimits"
+        ],
+        "summary": "Creates a new Group Quota request for a specific resources.",
```

Comment:
>Creates a new Group Quota request for a specific resources. [](http://example.com/codeflow?start=20&length=59)

The description doesn't match with the resource type "groupQuotaLimits"?

Reply:
@visingla-ms I've updated the description. Let me know if that needs further adjustment.